packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -556,7 +556,25 @@ const requestProcessor =
         })
         .build()
 
-module.exports = [client, ssr, renderer, clientOptional, requestProcessor]
+// TODO: can we make this one config with multiple entries? would that dedupe the dependencies?
+const extensions = (pkg.mobify?.app?.extensions || []).map((extension) => {
```

Comment:
nit: We might mix up the PWA Kit 'extensions' concept with the Webpack file 'extensions' concept: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/c6480c8f36029ff3770167ca41415db50913957c/packages/pwa-kit-dev/src/configs/webpack/config.js#L198
Do you think we should use something like 'pwaKitExtensions' instead?

Reply:
i'm happy to add comments to make it more readable

Reply:
done